src/Api/Invoice.php

```diff
@@ -157,6 +157,24 @@ public function create($contactId, $items, array $parameters = []): SevInvoice
         return $sevInvoice;
     }
 
+    // =========================== create reminder ====================================
+
+    /**
+     * Create invoice reminder.
+     *
+     * @param $invoiceId
+     * @return array
+     * @throws Exception
+     */
+    public function createReminder($invoiceId): array
+    {
+        return $this->_post(Routes::CREATE_REMINDER, [
+            'invoice' => [
+                'id' => $invoiceId,
+                "objectName" => "Invoice"
+            ]
+        ]);
+    }
 
```

Comment:
nice work. please add also documentation for this method. 
There is an invoice.md markdown file for English and German in the documentation folder.

Reply:
Wanna check again if it fits your criteria?